src/interactive-window/outputs/tracebackFormatter.ts

```diff
@@ -79,7 +79,7 @@ export class InteractiveWindowTracebackFormatter implements ITracebackFormatter
                 // We have a match, replace source lines first
                 const afterLineReplace = traceFrame.replace(LineNumberMatchRegex, (_s, prefix, num, suffix) => {
                     const n = parseInt(num, 10);
-                    const lineNumberOfFirstLineInCell = match!.line;
+                    const lineNumberOfFirstLineInCell = match!.hasCellMarker ? match!.line - 1 : match!.line;
```

Comment:
Not sure why I changed this before. This is what Don had originally and it works for me now.